lib/icedfrisby.js

```diff
@@ -1142,6 +1142,12 @@ class Frisby {
   }
 
 
+  _mochaTimeout () {
+    return (1 + this.current.retry) * this._timeout +
```

Comment:
I understand the next line, but why are we adding `retry * timeout` to that?

Reply:
This is the way I'm thinking of it.

If one retry is allowed, you need:

timeout interval for first request + retry backoff + timeout interval for second request

2 (timeout) + 1 (retry backoff)

Does that make sense?

Reply:
That does make sense, thanks.

Reply:
I'll add that explanation in a comment.

Reply:
Come to think of it, it seems clearer factored like this: 840c6c1.